components/button/examples/60-disabled-state.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { Button } from '../src';
+
+export default () => (
+	<>
+		<h3>
+			Disabled buttons with a <code>&lt;button&gt;</code> tag
+		</h3>
+		<p>
+			<Button appearance="primary" disabled>
+				Primary
+			</Button>{' '}
+			<Button appearance="hero" disabled>
+				Hero
+			</Button>{' '}
+			<Button appearance="neutral" disabled>
+				Neutral
+			</Button>{' '}
+			<Button appearance="faint" disabled>
+				Faint
+			</Button>{' '}
+			<Button appearance="link" disabled>
+				Link
+			</Button>
+		</p>
+		<p>
+			<Button appearance="primary" soft disabled>
+				Primary soft
+			</Button>{' '}
+			<Button appearance="hero" soft disabled>
+				Hero soft
+			</Button>{' '}
+			<Button appearance="neutral" soft disabled>
+				Neutral soft
+			</Button>{' '}
+			<Button appearance="faint" soft disabled>
+				Faint soft
+			</Button>
+		</p>
+
+		<hr />
+
+		<h3>
+			Disabled buttons with an <code>&lt;a&gt;</code> tag
+		</h3>
+		<p>
+			<Button href="#0" appearance="primary" className="disabled">
```

Comment:
For further discussion on whether a button as an 'a' tag can be disabled and if so we should pass disabled as a prop and handle via emotion rather than passing and targeting a className

Reply:
I think we should run with a `<a>` (link) cant be disabled in HTML, it's bad UX to do this too.

We have to be careful doing everything 'the React way' because we have to also maintain support for non-react environments via this new Legacy support layer (we're going to be adding down the track). I'd like to leverage default html attributes where available.